crytic_compile/utils/naming.py

```diff
@@ -151,7 +151,7 @@ def convert_filename(
     if not filename.is_absolute():
         filename = cwd.joinpath(filename)
 
-    absolute = filename
+    absolute = Path(os.path.abspath(filename))
```

Comment:
Key change is here, other changes are just minor additions/fixes to in-line docs